server/controllers/PerformancePost.resolver.ts

```diff
@@ -73,14 +73,12 @@ export default class PerformancePostResolver {
 
     @Query(() => [PerformancePost])
     public async getPerformancePostsCursor(@Arg('params') { limit, dateTimeCursor }: CursorPaginationInputs,
-        @Arg('filter', { nullable: true }) { states }: PerformancePaginationFiltersInput) {
+        @Arg('filters', { nullable: true }) { states }: PerformancePaginationFiltersInput = {}) {
         const where: WhereOptions = { createdAt: { [Op.lt]: dateTimeCursor } };
 
         if (states) {
-            where.states = {
-                state: {
-                    [Op.in]: states
-                }
+            where.state = {
+                [Op.in]: ['PUBLISHED']
```

Comment:
Кажется, так должно быть)
```suggestion
                [Op.in]: states
```